sdk/eventhub/event-hubs/test/public/utils/testInMemoryCheckpointStore.ts

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CheckpointStore, PartitionOwnership, Checkpoint } from "../../../src";
+import { generate_uuid } from "rhea-promise";
+
+/**
+ * The `EventProcessor` relies on a `CheckpointStore` to store checkpoints and handle partition
+ * ownerships. `InMemoryCheckpointStore` is simple partition manager that stores checkpoints and
+ * partition ownerships in memory of your program.
+ *
+ * You can use the `InMemoryCheckpointStore` to get started with using the `EventProcessor`.
+ * But in production, you should choose an implementation of the `CheckpointStore` interface that will
+ * store the checkpoints and partition ownerships to a durable store instead.
+ *
+ * @class
+ * @internal
+ * @ignore
+ */
+export class TestInMemoryCheckpointStore implements CheckpointStore {
```

Comment:
Let's change the class comment to say that, yes, this is a copy of the InMemoryCheckpointStore and that, yes, we did it on purpose and here's why.

Also, please make sure that @chradek  is listed as the person in charge of making that decision.